### PR TITLE
Extend "Ein Feld im Backend ausblenden"

### PR DESCRIPTION
**Pull request by fiedsch**
As commented in Slack the current example would remove the field while the title suggests it would only be hidden.

I tried to be a bit more specific to show both options (hiding and removing).

If we agree on the new description, I will extend the PR to also contain the english texts.

### CODE REVIEW

File: docs/manual/guides/dca-adjustments.de.md [hunk 1]
```diff
@@ -42,10 +42,38 @@ $GLOBALS['TL_DCA']['tl_news']['fields']['headline']['eval']['preserveTags'] = tr
...
+PaletteManipulator::create()
+    ->removeField('company')
+    ->applyToPalette('default', 'tl_member')
+;
+
 unset($GLOBALS['TL_DCA']['tl_member']['fields']['dateOfBirth']);
 ```
+Beachte: dadurch wird beim nächsten 
+[Aktualisieren der Datenbank](/de/installation/contao-installtool/#tabellen-aktualisieren) die Spalte `dateOfBirth` 
+gelöscht!
```
Comment (fritzmg) [inline docs/manual/guides/dca-adjustments.de.md]: Not by default. Any delete operation has to be specifically confirmed by the user.

File: docs/manual/guides/dca-adjustments.de.md [hunk 2]
```diff
@@ -42,10 +42,38 @@ $GLOBALS['TL_DCA']['tl_news']['fields']['headline']['eval']['preserveTags'] = tr
...
+use Contao\CoreBundle\DataContainer\PaletteManipulator;
+
+PaletteManipulator::create()
+    ->removeField('company')
+    ->applyToPalette('default', 'tl_member')
+;
+
+unset($GLOBALS['TL_DCA']['tl_member']['fields']['dateOfBirth']['eval']['feEditable']);
+unset($GLOBALS['TL_DCA']['tl_member']['fields']['dateOfBirth']['eval']['feViewable']);
+unset($GLOBALS['TL_DCA']['tl_member']['fields']['dateOfBirth']['eval']['feGroup']);
```
Comment (fritzmg) [inline docs/manual/guides/dca-adjustments.de.md]: This removes `dateOfBirth` from the member modules, but above you are removing `company`.

**Review by fritzmg — CHANGES_REQUESTED** [review]

File: docs/manual/guides/dca-adjustments.de.md (hunk 1 above)
Comment (fiedsch) [reply docs/manual/guides/dca-adjustments.de.md]: What about "zum Löschen vorgeschlagen".

File: docs/manual/guides/dca-adjustments.de.md (hunk 2 above)
Comment (fiedsch) [reply docs/manual/guides/dca-adjustments.de.md]: Thanks. Is a copy/paste error :-(

File: docs/manual/guides/dca-adjustments.de.md [hunk 3]
```diff
@@ -42,10 +42,38 @@ $GLOBALS['TL_DCA']['tl_news']['fields']['headline']['eval']['preserveTags'] = tr
...
+use Contao\CoreBundle\DataContainer\PaletteManipulator;
+
+PaletteManipulator::create()
+    ->removeField('company')
+    ->applyToPalette('default', 'tl_member')
+;
+
+unset($GLOBALS['TL_DCA']['tl_member']['fields']['company']);
 ```
+Beachte: dadurch wird beim nächsten
```
Comment (fritzmg) [inline docs/manual/guides/dca-adjustments.de.md:75]:
```suggestion

Beachte: dadurch wird beim nächsten 
```

**Review by fritzmg — CHANGES_REQUESTED** [review]

**Comment by fritzmg** [comment]
👍 LGTM

**Comment by fiedsch** [comment]
Thanks for the suggestions. I will add the english translation to the PR later this afternoon.

**Review by netzarbeiter — APPROVED** [review]
Thank you, Andreas

**Pull request merged**